content/docs/command-reference/stage/add.md

```diff
@@ -246,6 +246,17 @@ data science experiments.
 - `--desc <text>` - user description of the stage (optional). This doesn't  
   affect any DVC operations.
 
+- `--live <path>` - specify the directory `path` for
+  [Dvclive](/doc/dvclive/dvclive-with-dvc) to write logs in. Saved in the `live`
+  field of `dvc.yaml`.
+
+- `--live-no-cache <path>` - specify a Dvclive log file <abbr>output</abbr> (not
+  <abbr>cached</abbr>). Turns any `-o <path>` into a `-O`.
```

Comment:
Is that what this does? The cli help output isn't very clear

```
$ dvc stage add -h
...
  --live-no-cache <path>
                        Declare output as dvclive (do not put into DVC cache).
```

Reply:
"Specify a Dvclive log directory" rather than "file". Its the same as `live` but not cached.
I think we might also need to point that in case of `--live` - the `summary` JSON file still has to be versioned by git.

Reply:
Updated.

> we might also need to point that in case of --live - the summary JSON file still has to be versioned by git

Yes but not in this ref. as there's 0 context about Dvclive. The explanation only makes sense in the DVClive docs, I think. I'll check if it's mentioned there... ⌛

Reply:
Updated some more in f326fcb3.